tgui/packages/tgui-panel/settings/SettingsPanel.js

```diff
@@ -157,62 +152,41 @@ export const SettingsGeneral = (props, context) => {
         </LabeledList.Item>
       </LabeledList>
       <Divider />
-      <Box>
-        <Flex mb={1} color="label" align="baseline">
-          <Flex.Item grow={1}>Highlight text (comma separated):</Flex.Item>
-          <Flex.Item shrink={0}>
-            <ColorBox mr={1} color={highlightColor} />
-            <Input
-              width="5em"
-              monospace
-              placeholder="#ffffff"
-              value={highlightColor}
-              onInput={(e, value) =>
-                dispatch(
-                  updateSettings({
-                    highlightColor: value,
-                  })
-                )
-              }
+      <Button icon="save" onClick={() => dispatch(saveChatToDisk())}>
+        Save chat log
+      </Button>
+    </Section>
+  );
+};
+
+const TextHighlightSettings = (props, context) => {
+  const highlightSettings = useSelector(context, selectHighlightSettings);
+  const dispatch = useDispatch(context);
+  return (
+    <Section fill>
```

Comment:
```suggestion
    <Section fill scrollable>
```
otherwise you can add too many sections and it goes off the bottom of the window